### DISCUSSION TITLE
Ability to left click in the image window to scrub around

### DISCUSSION THREAD

**Discussion by g-steele** [videovillage/community#120] [Ideas]
Hi Screen Team,

Love the software and have been using it more every day (the playlists are great!). One feature that I use all of the time in RV is the ability to scrub the timeline by left clicking in the image window and scrubbing left or right for a more precise localized review. It beats having to find the playbar and jumping around. Let me know if you know that behavior and if not, I can email a video screengrab that shows how it works.

Thanks. 

-greg

**Comment by gregcotten** [comment]
Hi Greg! Glad to hear you're liking Screen. I think that's a great idea, although our left click and scrub actually moves around the window. What do you think about Shift-Left Click or something like that?

**Comment by g-steele** [comment]
Hi Greg. Thanks for your response! I see now that's the default behavior as we have the option of turning off the visibility of the movie or playlist title bar.

I'll take the shift key combo if it's not too burdensome of a request, but in a perfect world, I would rather forgo the need to hold down shift key and instead, give up the window moving functionality. :) 

Thanks, 

-greg
